src/vivarium_census_prl_synth_pop/results_processing/ssn_and_itin.py

```diff
@@ -228,6 +138,47 @@ def get_ssn_map(
     return ssn_map_dict
 
 
+def get_itin_map(
```

Comment:
I think `get_itin_map` and `get_ssn_map` are identical, except they read from a different artifact key. That being said, this won't be the case after you account for my comment in the other pr about the `ssn_id` column. Nevertheless, there seems to be scope to consolidate the code.